### DISCUSSION TITLE
Docker icons not rendering correctly

### DISCUSSION THREAD

**Discussion by sollymay** [nvim-tree/nvim-tree.lua#2190] [Q&A] [answered]
### Description

Looks like the docker icons are not displaying properly for some reason? All other icons work properly

<img width="1239" alt="Screenshot 2023-05-03 at 3 50 10 PM" src="https://user-images.githubusercontent.com/6496698/236031449-a07028bf-b009-4de9-a45b-5045efe6a48b.png">

### Neovim version

```text
NVIM v0.9.0
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 13.3.1

### nvim-tree version

commit 53295de

### Minimal config

```lua
local tree_cb = require 'nvim-tree.config'.nvim_tree_callback

local function open_nvim_tree(data)
    local directory = vim.fn.isdirectory(data.file) == 1

    if not directory then
        return
    end

    vim.cmd.cd(data.file)
    -- open the tree
    require("nvim-tree.api").tree.open()
end

vim.g.view_mappings_list = {
    ["<C-.>"] = tree_cb("edit_in_place"),
}

return {
    'nvim-tree/nvim-tree.lua',
    dependencies = { 'nvim-tree/nvim-web-devicons' },
    config = function()
        require("nvim-tree").setup({
            view = {
                width = 50,
                number = true,
                relativenumber = false,
            },
            git = {
                ignore = false,
            },
            actions = {
                open_file = {
                    quit_on_open = true
                },
                change_dir = {
                    enable = false,
                },
            },
```

### Steps to reproduce

open a project with a docker-compose.yml or Dockerfile

### Expected behavior

icons render correctly

### Actual behavior

icons render incorrectly

**Comment by gegoune** [comment, marked as answer]
Icons come from nvim-web-devicons and nvim-tree just prints them so unlikely that it is a culprit. Try some different font, update your current font, etc, and if none of it works issue should be transferred to nvim-web-devicons repo. 

I am assuming your font misses required glyph as all the others render fine.

**Comment by sollymay** [comment]
seems like updating the font was the solution to this problem! thank you for pointing in the right direction!
